src/docs/product/issues/index.mdx

```diff
@@ -32,6 +32,26 @@ From the **Issues** page, you can begin to triage. The page is organized into ta
 
 Learn more about triaging issues and their different states in [Issue States and Triage](/product/issues/states-triage/).
 
+### Display Options
+
+You can use the display dropdown in the issues page to see error counts as a % of [sessions](/product/releases/health/#session).
+
+![Issues display dropdown](issues-events-as-percentage-dropdown.png)
+
+For this option to be enabled you need to have selected a project that has [release health set up](/product/releases/health/setup) and need to have issues and sessions present in the selected period.
+
+With `Events as %` option selected, instead of event counts you will see the issues' events as a % of overall sessions for the project.
```

Comment:
```suggestion
With the `Events as %` option selected, instead of event counts, you will see the issues' events as a percentage of overall sessions for the project.
```